lib/gssapi/mech/context.h

```diff
@@ -31,6 +31,7 @@
 
 struct _gss_context {
 	gssapi_mech_interface	gc_mech;
+	gssapi_mech_interface	gc_mech_actual;
```

Comment:
`gc_mech_actual` doesn't appear to get used anywhere, I agree it might be nice to have but do we need it?

Reply:
I can remove it.  My idea had been to also implement a `GM_USE_MG_CTX` for when SPNEGO returns `GSS_S_COMPLETE`: then the context returned by SPNEGO would replace the context output to the application.  In fact, we don't need `gc_mech_actual` for that anyways, so I should remove it.